src/NServiceBus.Hosting.Windows/WindowsHost.cs

```diff
@@ -32,9 +31,10 @@ public void Start()
             {
                 genericHost.Start().GetAwaiter().GetResult();
             }
-            catch (Exception ex)
+            catch (Exception exception)
```

Comment:
Not against that in general but I tend to not apply refactorings in patches to keep diffs as clean as possible.